core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -142,6 +144,17 @@ angular.module('oppia').directive('adminMiscTab', [
           ctrl.dataExtractionQueryStatusMessage = message;
         };
 
+
+        ctrl.sendDummyMailToAdmin = function() {
+          $http.post(SEND_DUMMY_MAIL_HANDLER_URL)
+            .then(function(response) {
+              ctrl.setStatusMessage(response.data.msg);
```

Comment:
Note that you can still show a status message (and it's a good idea!), but you don't need to pass it in from the backend. You can just define it here.

Reply:
Done.